src/backend/tests/test_friendly_rule.py

```diff
@@ -1,14 +1,38 @@
+from dateutil.relativedelta import relativedelta
+
 from expungeservice.expunger import Expunger
 from expungeservice.models.expungement_result import EligibilityStatus
 from expungeservice.models.record import Record
 from tests.factories.case_factory import CaseFactory
 from tests.factories.charge_factory import ChargeFactory
-from tests.factories.expunger_factory import ExpungerFactory
 from tests.time import Time
+from datetime import date
+
+
+def test_eligible_mrc_with_single_arrest():
+    three_yr_mrc = ChargeFactory.create(disposition=["Convicted", Time.THREE_YEARS_AGO])
+
+    arrest = ChargeFactory.create(disposition=["Dismissed", Time.THREE_YEARS_AGO])
+
+    case = CaseFactory.create()
+    case.charges = [three_yr_mrc, arrest]
+    record = Record([case])
+    expunger = Expunger(record)
+
+    expunger.run()
+    assert arrest.expungement_result.time_eligibility.status is EligibilityStatus.ELIGIBLE
+    assert (
+        arrest.expungement_result.time_eligibility.reason
+        == "The friendly rule: time eligibility of the arrest matches time eligibility of the conviction."
+    )
+    assert arrest.expungement_result.time_eligibility.date_will_be_eligible == date.today()
+
+    assert three_yr_mrc.expungement_result.time_eligibility.status is EligibilityStatus.ELIGIBLE
+    assert three_yr_mrc.expungement_result.time_eligibility.reason == ""
+    assert three_yr_mrc.expungement_result.time_eligibility.date_will_be_eligible == date.today()
 
 
 def test_arrest_is_unaffected_if_conviction_is_older():
```

Comment:
This test name is really confusing. what's the actual thing being tested here? Would the arrest be affected if the conviction was newer?